frontend/src/metabase/lib/formatting.js

```diff
@@ -740,6 +740,18 @@ export function formatValueRaw(value: Value, options: FormattingOptions = {}) {
 
   if (value == null) {
     return null;
+  } else if (
+    options.click_behavior &&
+    clickBehaviorIsValid(options.click_behavior) &&
+    options.jsx
+  ) {
+    // Style this like a link if we're in a jsx context.
+    // It's not actually a link since we handle the click differently for dashboard and question targets.
```

Comment:
Just FYI for any readers, I think these could be actual links but there's a bit more nuance in generating the URLs. We need to make sure that we've loaded data about the target dashboard/question and have a way to encode GUI question parameters in the URL.